include/psen_scan_v2/monitoring_frame_deserialization.h

```diff
@@ -24,34 +24,17 @@
 
 namespace psen_scan_v2
 {
+namespace monitoring_frame
+{
 static constexpr uint32_t DEFAULT_DEVICE_STATUS{ 0 };
 static constexpr uint32_t OP_CODE_MONITORING_FRAME{ 0xCA };
 static constexpr uint32_t ONLINE_WORKING_MODE{ 0x00 };
 static constexpr uint32_t GUI_MONITORING_TRANSACTION{ 0x05 };
 static constexpr uint16_t NUMBER_OF_BYTES_SCAN_COUNTER{ 4 };
 static constexpr uint16_t NUMBER_OF_BYTES_SINGLE_MEASURE{ 2 };
 static constexpr uint16_t NUMBER_OF_BYTES_SINGLE_INTENSITY{ 2 };
-class MonitoringFrameAdditionalFieldHeader
-{
-public:
-  using Id = uint8_t;
-  using Length = uint16_t;
-
-public:
-  MonitoringFrameAdditionalFieldHeader(Id id, Length length);
-
-public:
-  Id id() const;
-  Length length() const;
-
-  static std::string idToString(Id id);
-
-private:
-  Id id_;
-  Length length_;
-};
 
-class MonitoringFrameFixedFields
+class FixedFields
```

Comment:
I know where the idea comes from but to stay consistent with the additional fields, I suggest to rename it to:
```suggestion
class Header
```
Because if each additional field has a header describing whats coming next, so does the MonitoringFrame itself.
Using this idea, we write for example:
```
using namespace psen_scan_v2;
const monitoring_frame::Header monitoring_frame_header;
const monitoring_frame::additional_field::Header additional_field_header;
```
This is more consistent in my opinion. What do you think?

Reply:
At the beginning I called it also header with exact the same idea...

@agutenkunst @martiniil complainted about that.  

To be consistent with the protocol description FixedFields would be better, because MonitoringFrame has no header.

Reply:
Yes because there is no such thing as a Header mentioned in the protocol manual. Only Headers for the Additional Fields. Therefore we concluded that FixedFields is the better naming to avoid confusion.